openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -52,6 +52,15 @@ def add_credit_course(self, enabled=True):
 
         return credit_course
 
+    def add_credit_eligible_course(self, credit_course, username):
```

Comment:
Actually, this method isn't really necessary. Just call `CreditEligibility.objects.create(course=credit_course, username=username, provider=CreditProvider.objects.get(provider_id=self.PROVIDER_ID))` in `test_is_user_eligible_for_credit`.